test/cases/ostree-simplified-installer.sh

```diff
@@ -946,145 +953,145 @@ sudo ansible-playbook -v -i "${TEMPDIR}"/inventory \
-# Get image id to run image
-EDGE_IMAGE_ID=$(sudo podman images --filter "dangling=true" --format "{{.ID}}")
-sudo podman run -d --name rhel-edge --network edge --ip "$STAGE_REPO_ADDRESS" "$EDGE_IMAGE_ID"
-# Wait for container to be running
-until [ "$(sudo podman inspect -f '{{.State.Running}}' rhel-edge)" == "true" ]; do
-    sleep 1;
-done;
-
-# Pull rebase commit to prod mirror
-greenprint "⛓ Pull rebase commit to prod mirror"
-sudo ostree --repo="$PROD_REPO" pull --mirror edge-stage "$OSTREE_REF"
-
-# Get ostree commit value.
-greenprint "🕹 Get ostree rebase commit value"
-REBASE_HASH=$(curl "${PROD_REPO_URL}/refs/heads/${OSTREE_REF}")
-
-# Clean compose and blueprints.
-greenprint "🧽 Clean up rebase blueprint and compose"
-sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
-sudo composer-cli blueprints delete rebase > /dev/null
-
-greenprint "🗳 Rebase ostree image/commit"
-sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" admin@${EDGE_GUEST_ADDRESS} "echo '${EDGE_USER_PASSWORD}' |sudo -S rpm-ostree rebase ${REF_PREFIX}:${OSTREE_REF}"
-sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" admin@${EDGE_GUEST_ADDRESS} "echo '${EDGE_USER_PASSWORD}' |nohup sudo -S systemctl reboot &>/dev/null & exit"
-
-# Sleep 10 seconds here to make sure vm restarted already
-sleep 10
-
-# Check for ssh ready to go.
-greenprint "🛃 Checking for SSH is ready to go"
-# shellcheck disable=SC2034  # Unused variables left for readability
-for _ in $(seq 0 30); do
-    RESULTS="$(wait_for_ssh_up $EDGE_GUEST_ADDRESS)"
-    if [[ $RESULTS == 1 ]]; then
-        echo "SSH is ready now! 🥳"
-        break
-    fi
-    sleep 10
-done
-
-# Check ostree rebase result
-check_result
-
-# Add instance IP address into /etc/ansible/hosts
-sudo tee "${TEMPDIR}"/inventory > /dev/null << EOF
-[ostree_guest]
-${EDGE_GUEST_ADDRESS}
-
-[ostree_guest:vars]
-ansible_python_interpreter=/usr/bin/python3
-ansible_user=${ANSIBLE_USER}
-ansible_private_key_file=${SSH_KEY}
-ansible_ssh_common_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
-ansible_become=yes
-ansible_become_method=sudo
-ansible_become_pass=${EDGE_USER_PASSWORD}
-EOF
-
-# Test IoT/Edge OS
-sudo ansible-playbook -v -i "${TEMPDIR}"/inventory \
-    -e image_type=rhel-edge \
-    -e ostree_commit="${REBASE_HASH}" \
-    -e skip_rollback_test="true" \
-    -e edge_type=edge-simplified-installer \
-    -e fdo_credential="true" \
-    -e sysroot_ro="$SYSROOT_RO" \
-    -e mfg_guest_int_name="${MFG_GUEST_INT_NAME}" \
-    -e fips="${FIPS}" \
-    /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
-
-check_result
+# ##################################################################
+# ##
+# ## Build rebased ostree repo
+# ##
+# ##################################################################
+# tee "$BLUEPRINT_FILE" > /dev/null << EOF
+# name = "rebase"
+# description = "An rebase rhel-edge container image"
+# version = "0.0.2"
+# modules = []
+# groups = []
```

Comment:
Any reason to keep all of this around and commented instead of removing it?

Reply:
Thanks, I will remove it.